test/fixtures/input/connect/config.yaml

```diff
@@ -9,5 +9,21 @@
   "tasks.max": "1",
   "time.interval": "HOURLY",
   "topics": "apples",
-  "connector.class": "AzureBlobSink"
+  "connector.class": "AzureBlobSink",
```

Comment:
This seems to be input file for CreateORUpdateConnector and offsets field is not supported in this API. Can you please check where is this file being used?

Reply:
for CreateOrUpdate, we are not passing the offset.